### ISSUE TITLE
Cannot execute drush uli with a parameter

### ISSUE THREAD

**Issue by oadaeh** [backdrop-contrib/drush#154]
When executing drush uli with a parameter, such as --uid, an error is generated:
> Error: Call to undefined function _drush_user_get_users_from_options_and_arguments() in backdrop_command_user_login() (line 85 of /usr/local/src/drush/commands/backdrop/commands/user/backdrop_user.drush.inc).

The _drush_user_get_users_from_options_and_arguments() is called within backdrop_command_user_login() in commands/user/backdrop_user.drush.inc.

The command was removed from Drush in this commit:
```
commit 13f2eaf14e136e857cc2d7223bef36b461898290
Author: Moshe Weitzman <weitzman@tejasa.com>
Date:   Thu Aug 21 14:37:57 2014 -0400

    Restructure user commands for OO
    
    Similar to the SQL and Role commands, the User commands now use a UserSingle object with methods like cancel(), block(), etc.
```

I tried just copying back in the missing functions, but the rabbit hole just kept getting deeper. Also, it does not look like the code it was replaced with is in the Backdrop version of Drush.

-----

PR: https://github.com/backdrop-contrib/drush/pull/156

**Comment by serundeputy** [comment]
Thanks @oadaeh 

I had not been a user of the `--name`, `--uid` and `--mail` options before!
I've filed a PR https://github.com/backdrop-contrib/drush/pull/156 to respect these options.

If you are up for testing that would be great.


thanks,
~Geoff

**Comment by serundeputy** [comment]
Merged #159.
